src/libcore/intrinsics.rs

```diff
@@ -44,6 +44,10 @@
 
 use marker::Sized;
 
+#[cfg(stage0)] pub use self::copy_memory as copy;
```

Comment:
I personally very much prefer to not use `// SNAP` as in the past it has ended up causing lots of pain when registering a new snapshot. It simultaneously forces the snapshot-maker to do the work necessary to remove it (which can be quite large and isn't always necessary) and also causes extra pain if a `// SNAP` annotation was added *after* the new snapshot was made.